tests/conftest.py

```diff
@@ -22,30 +24,22 @@ def pytest_configure(config):
 
 
 @pytest.fixture
-def phantomjs_driver(capabilities, driver_path, port):
-    """
-    Overrides default `phantomjs_driver` driver from pytest-selenium.
-
-    Default implementation uses ephemeral ports just as our tests but
-    it doesn't provide any way to configure them, for this reason we basically
-    recreate the driver fixture using port fixture.
-    """
-    kwargs = {}
-    if capabilities:
-        kwargs['desired_capabilities'] = capabilities
-    if driver_path is not None:
-        kwargs['executable_path'] = driver_path
-
-    kwargs['port'] = port.get()
-
-    from selenium.webdriver import PhantomJS
-
-    return PhantomJS(**kwargs)
+def chrome_options(chrome_options: ChromeOptions) -> ChromeOptions:
+    chrome_options.add_argument("--no-sandbox")
```

Comment:
As i commented before, i believe that it is not necessary.

Reply:
qt<->js test will fail to load the page on ci if this is not defined.